Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_True_Positive_Incident_Handling.yml

```diff
@@ -963,7 +977,7 @@ tasks:
           simple: Confirm which File Path will be deleted
         options: []
         optionsarg:
-        - simple: ${incident.processpaths}
+        - simple: ${inputs.FilePaths}
```

Comment:
Same as the comment above (unique transformer)

Reply:
Done.